docs/celestia-architecture/README copy.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
is the filename intentionally `README copy.md`?

Reply:
this was a remnant of the old master file, I'm assuming no, and consolidated the readmes here https://github.com/celestiaorg/celestia-core/pull/737/commits/a544403ce4741d720792c09582d73b5b098edf2e